docs/lakehouse-iceberg-catalog.md

```diff
@@ -33,11 +33,18 @@ Builds with Hadoop 2.10.x, there may be compatibility issues when accessing Hado
-| `catalog-backend` | Catalog backend of Gravitino Iceberg catalog. Supports `hive` or `jdbc` or `rest`.                                                                                                              | (none)        | Yes      | 0.2.0         |
-| `uri`             | The URI configuration of the Iceberg catalog. `thrift://127.0.0.1:9083` or `jdbc:postgresql://127.0.0.1:5432/db_name` or `jdbc:mysql://127.0.0.1:3306/metastore_db` or `http://127.0.0.1:9001`. | (none)        | Yes      | 0.2.0         |
-| `warehouse`       | Warehouse directory of catalog. `file:///user/hive/warehouse-hive/` for local fs or `hdfs://namespace/hdfs/path` for HDFS.                                                                      | (none)        | Yes      | 0.2.0         |
+| Property name                                      | Description                                                                                                                                                                                     | Default value | Required                                                    | Since Version |
+|----------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------|-------------------------------------------------------------|---------------|
+| `catalog-backend`                                  | Catalog backend of Gravitino Iceberg catalog. Supports `hive` or `jdbc` or `rest`.                                                                                                              | (none)        | Yes                                                         | 0.2.0         |
+| `uri`                                              | The URI configuration of the Iceberg catalog. `thrift://127.0.0.1:9083` or `jdbc:postgresql://127.0.0.1:5432/db_name` or `jdbc:mysql://127.0.0.1:3306/metastore_db` or `http://127.0.0.1:9001`. | (none)        | Yes                                                         | 0.2.0         |
+| `warehouse`                                        | Warehouse directory of catalog. `file:///user/hive/warehouse-hive/` for local fs or `hdfs://namespace/hdfs/path` for HDFS.                                                                      | (none)        | Yes                                                         | 0.2.0         |
+| `authentication.type`                              | The type of authentication for Iceberg catalog backend, currently we only support `kerberos`, `simple`.                                                                                         | `simple`      | No                                                          | 0.6.0         |
```

Comment:
Do we have this config option? Hive doesn't have this property now.

Reply:
I have added this to the Iceberg Hive backend configuration item.

Reply:
Could you align to the config option of Hive?

Reply:
Sure, let me check whether we need to check Hive or Iceberg for this point to align the configuration items.

Reply:
Why is Hive special?

Reply:
We haven't added the configuration for `Hive` yet. If it's necessary, I'll add it.